#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From 640d670b9797fd9e00935d8c64816ef4c2b2f4d5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ingo=20Kl=C3=B6cker?= <dev@ingo-kloecker.de>
Date: Mon, 2 Oct 2023 10:57:51 +0200
Subject: [PATCH 1/3] Set the modes on the date picker

This seems to have been forgotten when this was refactored.
---
 src/kdatecombobox.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/kdatecombobox.cpp b/src/kdatecombobox.cpp
index 6817922e..cfaaa83e 100644
--- a/src/kdatecombobox.cpp
+++ b/src/kdatecombobox.cpp
@@ -124,6 +124,7 @@ void KDateComboBoxPrivate::initDateWidget()
     if (m_options & KDateComboBox::DateKeywords) {
         modes |= KDatePickerPopup::Words | KDatePickerPopup::NoDate;
     }
+    m_dateMenu->setModes(modes);
 }

 void KDateComboBoxPrivate::updateDateWidget()
--
2.42.0
